### PR TITLE
New Port: Dark Sheep

### PR DESCRIPTION
**Pull request by mhoogcarspel**
# New Port for Dark Sheep

## Game Information
- **Title**: Dark Sheep
- **URL**: https://daisy-games.itch.io/dark-sheep

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [ ] muOS
- [X] KNULLI

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/darksheep/Dark Sheep.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+CONFDIR="$GAMEDIR/conf/"
+
+mkdir -p "$GAMEDIR/conf"
+export XDG_DATA_HOME="$CONFDIR"
+
+cd $GAMEDIR
```
Comment (Slayer366) [inline ports/darksheep/Dark Sheep.sh:31]: The script already enters the game directory at line 26.  While harmless, it only needs to do this once.
```suggestion
```

File: ports/darksheep/darksheep/darksheep.gptk [hunk 2]
```diff
@@ -0,0 +1,27 @@
+back = esc
+start = enter
+
+a = space
+b = backspace
+x = r
+y = "\
```
Comment (Slayer366) [inline ports/darksheep/darksheep/darksheep.gptk:7]:
```suggestion
y = \"

```

File: ports/darksheep/darksheep/darksheep.gptk [hunk 3]
```diff
@@ -0,0 +1,27 @@
+back = esc
+start = enter
+
+a = space
+b = backspace
+x = r
+y = "\
+
+l1 = "\
```
Comment (Slayer366) [inline ports/darksheep/darksheep/darksheep.gptk:9]:
```suggestion
l1 = \"

```

File: ports/darksheep/darksheep/darksheep.gptk [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+a = space
+b = backspace
+x = r
+y = "\
+
+l1 = "\
+l2 = enter
+r1 = "\
+r2 = "\
```
Comment (Slayer366) [inline ports/darksheep/darksheep/darksheep.gptk:12]:
```suggestion
r1 = \"
r2 = \"

```

File: ports/darksheep/darksheep/darksheep.gptk [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+left_analog_up = up
+left_analog_down = down
+left_analog_left = left
+left_analog_right = right
+
+right_analog_up = "\
+right_analog_down = "\
+right_analog_left = "\
+right_analog_right = "\
```
Comment (Slayer366) [inline ports/darksheep/darksheep/darksheep.gptk:27]:
```suggestion
right_analog_up = \"
right_analog_down = \"
right_analog_left = \"
right_analog_right = \"
```

**Review by Slayer366 — CHANGES_REQUESTED** [review]

File: ports/darksheep/port.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Dark Sheep",
+    "porter": [
+      "Kompaan"
+    ],
+    "desc": "Dark Sheep is a block-pushing puzzle game with horror elements, and a love letter to the Commodore 64, both in visuals and audio! Plays like classic Sokoban with new twists that force you to find creative solutions!",
+    "desc_md": null,
+    "inst": "Grab the windows version from https://daisy-games.itch.io/dark-sheep\nand copy DarkSheep.pck into /roms/ports/darksheep/gamedata\n\nText is harder to read on 320p, but playable. Works best on 480p and up.",
```
Comment (Slayer366) [inline ports/darksheep/port.json:16]: muOS has the ports placed in the ports folder at the root which in turn mounts as /mnt/mmc/ports for the 1st memory stick or /mnt/sdcard/ports for the 2nd memory stick so /roms/ports (which could also be /roms2/ports for the 2nd memory stick on ArkOS, for example) isn't absolute.
```suggestion
    "inst": "Grab the windows version from https://daisy-games.itch.io/dark-sheep\nand copy DarkSheep.pck into ports/darksheep/gamedata\n\nText is harder to read on 320p, but playable. Works best on 480p and up.",
```

**Review by Slayer366 — CHANGES_REQUESTED** [review]

**Comment by mhoogcarspel** [comment]
Thanks for the second pair of eyes Slayer!

**Review by Slayer366 — APPROVED** [review]

**Pull request merged**